### DISCUSSION TITLE
What is the tech term for a private network - inaccesible to outsiders - that allows internal users to communicate?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#316] [Galaxy Trivia] [answered]
**Hint:** not VPN

**Comment by thuongtruong1009** [comment, marked as answer]
The tech term for a private network that allows internal users to communicate and is inaccessible to outsiders is "intranet". An intranet is a private network that is designed to serve the internal communications needs of an organization, such as sharing documents, collaborating on projects, and communicating between departments or teams. Unlike the internet, which is a public network accessible to anyone with an internet connection, an intranet is only accessible to authorized users within the organization, making it a more secure way to share information and collaborate on projects.
